identity-account/src/account/config.rs

```diff
@@ -35,23 +35,19 @@ impl AccountSetup {
 
 /// Configuration for [`Account`][crate::account::Account]s.
 #[derive(Clone, Debug)]
-pub(crate) struct AccountConfig {
-  pub(crate) autosave: AutoSave,
-  pub(crate) autopublish: bool,
-  pub(crate) testmode: bool,
-  pub(crate) milestone: u32,
+pub struct AccountConfig {
+  pub autosave: AutoSave,
+  pub autopublish: bool,
+  pub testmode: bool,
 }
```

Comment:
Making `AccountConfig` public means any changes to it will be breaking, and has the side-effect of exposing the `testmode` field which should only be used when compiled for tests, never seen by end-developers.

Does this absolutely need to be `public`? If it does, maybe conditionally compile `testmode`? (Unsure if this code works)

```suggestion
pub struct AccountConfig {
  pub autosave: AutoSave,
  pub autopublish: bool,
  #[cfg(test)]
  pub testmode: bool,
}
```

Reply:
I changed the implementation to use the Builder instead of `AccountConfig`.